Libraries/DXUT/Core/DXUT.cpp

```diff
@@ -35,9 +35,9 @@ class DXUTLock
 {
 public:
 #pragma prefast( suppress:26166, "g_bThreadSafe controls behavior" )
-    inline _Acquires_lock_(g_cs) DXUTLock()  { if( g_bThreadSafe ) EnterCriticalSection( &g_cs ); }
```

Comment:
do you know why we have changes here? do we want to take these?

Reply:
Oh this is just the change in formatting so we're totally safe here. I'm so annoyed at keeping using "( param1, param2 )..." style so I used VS format function to change it :)